packages/cache/src/lib/CacheMiddleware.ts

```diff
@@ -122,7 +130,7 @@ export class CacheMiddleware<StoredValue = unknown> extends JoshMiddleware<Cache
   @PreProvider()
   public async [Method.Every](payload: Payloads.Every<StoredValue>): Promise<Payloads.Every<StoredValue>> {
     const { provider: cache } = this.context;
-    const { type, hook, path, value } = payload;
+    const { type, value } = payload;
```

Comment:
```suggestion
    const { type } = payload;
```
Since this is not being used in the `Payloads.Every.ByHook` scope it should be moved to the `Payloads.Every.ByValue` scope.